SU-SWS/linky_clicky
# DEVOPS-256

# READY FOR REVIEW

# Summary
- Reinstate the stanford_capx Feature
- Add a new step definition for the cache-expiration Feature

# Needed By (Date)
- When does this need to be merged by? Meh.

# Criticality
- How critical is this PR on a 1-10 scale? 2/10
- Meh. Fixes annoyances for me, and could prevent future breakage.

# Steps to Test

1. Check out this branch
2. `git-crypt unlock /path/to/key`
3. Create a D7 site with the `stanford` profile and run `sites/uat/features/high-value/stanford_capx.feature`
4. Create a JSA site and run `products/jsa/features/cache-expiration.feature
5. Enjoy

# Affected Projects or Products
- D7 sites

# Associated Issues and/or People
- JIRA ticket: DEVOPS-256

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
